RGB[] btColors
int8 blinkyID
---
int8 success
